### ISSUE TITLE
Intermittent PHP Warning

### ISSUE THREAD

**Issue by alaczek** [Automattic/components.underscores.me#82]
Today I saw the following PHP Warning at the very bottom of the site:

> Oh no! I couldn't open the zip: /home/componen/public_html/prototype/theme-pattern-library-types-blog-traditional.zip

Despite the warning I was able to generate a theme. When I refreshed the page after approx. 5 minutes, the warning was gone. Has anyone else encountered this? 
Needs more investigation.

**Comment by sarahmonster** [comment]
I saw this yesterday, then it disappeared, so I figured I'd hallucinated it. It cropped up again today. 

I've seen it both locally:
![screen shot 2016-01-29 at 12 06 44 pm](https://cloud.githubusercontent.com/assets/376315/12675280/fcada374-c680-11e5-9a71-4cf7b6102244.png)

and on the production site:
![screen shot 2016-01-29 at 12 08 59 pm](https://cloud.githubusercontent.com/assets/376315/12675291/18cae4ea-c681-11e5-857a-eda4350e5dab.png)

Seems intermittently, although a refresh clears it up—I wonder if it happens when the site hasn't been loaded in a while and it's related to the time delay in generating the zips? Just spitballing.
